src/greeklt/remove_accentuation.py

```diff
@@ -1,4 +1,4 @@
-def remove_accentuation(string: str):
+def remove_accentuation(string: str, modifier=0):
```

Comment:
It should be modifier=True not 0. I also would like to rename modifier to something like keep_dieresis or something like that.